client/my-sites/term-tree-selector/docs/example.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+
+/**
+ * Internal dependencies
+ */
+import TermTreeSelector from '../';
+import observe from 'lib/mixins/data-observe';
+import sites from 'lib/sites-list';
+
+const TermTreeSelectorExample = React.createClass( {
+	mixins: [ observe( 'sites' ), PureRenderMixin ],
+
+	render() {
+		const primary = this.props.sites.getPrimary();
+
+		/* eslint-disable wpcalypso/jsx-classname-namespace */
+		return (
+			<div className="design-assets__group">
+				<h2>
+					<a href="/devdocs/app-components/term-tree-selector">Term Tree Selector</a>
+				</h2>
+				<div style={ { width: 300 } }>
+					{ this.props.sites.initialized && (
+						<TermTreeSelector
+							siteId={ primary ? primary.ID : 3584907 }
+							taxonomy="category" />
+					) }
+				</div>
+			</div>
+		);
+		/* eslint-enable wpcalypso/jsx-classname-namespace */
+	}
+} );
+
+export default React.createClass( {
+	displayName: 'TermTreeSelector',
+
+	mixins: [ PureRenderMixin ],
```

Comment:
I think it's good to keep as-is since it's consistent with the component above (where mixins are needed) and avoids an extra import, but FYI there's `PureComponent` option from `react-pure-render`:

```js
import PureComponent from 'react-pure-render/component';

export default class TermTreeSelector extends PureComponent {
    /* ... */
}
```